OWML.Common/IModInputHandler.cs

```diff
@@ -3,6 +3,7 @@
     public interface IModInputHandler
     {
         IModInputCombination RegisterCombination(IModBehaviour mod, string name, string combination);
+        IModInputTextures Textures { get; }
```

Comment:
Put this at the top to separate fields from methods?

Reply:
yeah, you are probably right. Sorry, Alek

Reply:
don't be sorry! :P